aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -314,6 +314,29 @@ Tensor pow_scalar_Tensor_batching_rule(Scalar other, const Tensor& self) {
   return makeBatched(output_physical, BatchDims(old_bdims.begin(), old_bdims.end()));
 }
 
+// I am quite sad that we need to register operators with exploded TensorOptions,
```

Comment:
cc @smessmer

Reply:
In the terminal state, all of the `native::` implementations should also be using exploded tensor options, so you no longer need to boilerplate it